### ISSUE TITLE
Exception during autocomplete

### ISSUE THREAD

**Issue by Necavi** [Talv/vscode-sc2-galaxy#1]
I was writing out UserDataGetFixed() and upon adding parenthesis I experience this exception. Running the latest versions of VSCode and the galaxy extension.

```
[Error - 6:38:53 PM] [TypeError] el.getParameters is not a function
TypeError: el.getParameters is not a function
    at S2WorkspaceMetadata.getFunctionArgumentsDoc (C:\Users\Necavi\.vscode\extensions\talv.sc2galaxy-1.7.3\node_modules\plaxtony\lib\service\s2meta.js:176:32)
    at SignaturesProvider.getSignatureOfFunction (C:\Users\Necavi\.vscode\extensions\talv.sc2galaxy-1.7.3\node_modules\plaxtony\lib\service\signatures.js:53:71)
    at SignaturesProvider.getSignatureAt (C:\Users\Necavi\.vscode\extensions\talv.sc2galaxy-1.7.3\node_modules\plaxtony\lib\service\signatures.js:100:40)
    at Server.<anonymous> (C:\Users\Necavi\.vscode\extensions\talv.sc2galaxy-1.7.3\node_modules\plaxtony\lib\service\server.js:396:44)
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\Necavi\.vscode\extensions\talv.sc2galaxy-1.7.3\node_modules\plaxtony\lib\service\server.js:10:58)
    at <anonymous>
### onSignatureHelp | 7.456ms
[Error - 6:38:53 PM] Request textDocument/signatureHelp failed.
  Message: Request textDocument/signatureHelp failed with message: el.getParameters is not a function
  Code: -32603 
```

**Comment by Talv** [comment]
Thanks for the report. It comes from using `ArgumentsDefault` in `sc2galaxy.completion.functionExpand` config. The feature itself wasn't quite finalized. `ArgumentsNull` is the best verbose and stable config for the time being.

Hopefully it'll be fixed next release https://github.com/Talv/plaxtony/issues/6

**Comment by Talv** [comment]
Ops, that's not because of code completions. Got a similar report regarding `UserData` functions, initially thought it was related to that config.

But seems it completely fails at them, even when it's just `signatureHelp` request.

**Comment by Necavi** [comment]
Ack! Yeah, my report was slightly inaccurate. You are correct that it is in the signature help not in autocomplete. I was counting them as the same thing for some odd reason.

**Comment by Talv** [comment]
Should be fixed in `v1.7.6`
